src/components/pages/HomePage/HomePage.tsx

```diff
@@ -6,6 +6,14 @@ import PageWrapper from "../PageWrapper/PageWrapper";
 const HomePage: React.FC = () => {
   const auth = useAuth();
 
+  if (auth.currentProjectId === undefined) {
+    return (
+      <PageWrapper title={`Welcome ${auth.user?.full_name}`}>
```

Comment:
The `PageWrapper` component should wrap the whole return. It's repeated twice here ;)